base/docs/basedocs.jl

```diff
@@ -3186,13 +3186,23 @@ Any
     Union{}
 
 `Union{}`, the empty [`Union`](@ref) of types, is the type that has no values. That is, it has the defining
-property `isa(x, Union{}) == false` for any `x`. `Base.Bottom` is defined as its alias and the type of `Union{}`
-is `Core.TypeofBottom`.
+property `isa(x, Union{}) == false` for any `x`. Thus it is an *empty*/*uninhabited* type.
```

Comment:
I would switch the order of these two paragraphs in the docstring. There are other uninhabited types, but `Union{}` is the unique lattice bottom. So put that first an then say: "In particular, this implies..."

Reply:
Is it OK now?